specialization_week_1/specialization_week_1.py

```diff
@@ -0,0 +1,115 @@
+"""
+TASK 1
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+"""
+
+class CashRegister:
+
+    def __init__(self):
+        self.total_items = {}
+
+    def add_item(self, item, price):
+        self.item = item
```

Comment:
⚠️ `self.item` and `self.price` represent an individual price of an item which shouldn't be a direct member of the class. It should be tracked inside the `total_items` dictionary